app/src/org/commcare/views/widgets/MediaWidget.java

```diff
@@ -147,7 +147,9 @@ private SecretKeySpec getSecretKey() {
     }
 
     protected void togglePlayButton(boolean enabled) {
-        mPlayButton.setEnabled(enabled);
+        if (mPlayButton != null) {
```

Comment:
curious if you are running into an error here ? My assumption is that this method only gets called for widgets with play button.